app/services/google/scrape_service.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Google
+  class ScrapeService
+    def initialize(keyword_id:)
+      @keyword_id = keyword_id
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    def call!(search_service: nil)
+      keyword = Keyword.find_by id: keyword_id
+      raise_keyword_not_found unless keyword
+      @keyword = keyword
+
+      begin
+        search_service ||= Google::SearchService.new(keyword.content)
+        html = search_service.search!
+      rescue GoogleScraperRuby::Errors::SearchServiceError => e
```

Comment:
Avoid [abbreviated variable names](https://nimblehq.co/compass/development/code-conventions/#convention-12):

```suggestion
      rescue GoogleScraperRuby::Errors::SearchServiceError => error
```

Prefer spelling out every variable name. You are not saving many keystrokes 😉  and it helps readability in the long run.

Reply:
Fixed in 19712c4f4945fdd0cb24eb04d37db0d532b43fa5